homeassistant/brands/artsound.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "artsound",
+  "name": "ArtSound",
+  "integrations": ["artsound"]
+}
```

Comment:
Brands only count for brands with 2+ integrations

Reply:
I didn't see the requirement listed on https://developers.home-assistant.io/docs/creating_integration_brand and had modeled it after another brand with one virtual integration.

Reply:
Which one? It could be that that one is a protocol brand which has to do with WWHA

Reply:
Yes, it's listed under the WWHA program. I'll check if ArtSound will apply for this too.